source/atomics.tex

```diff
@@ -238,71 +238,71 @@
 to all other atomic accesses to that object. \end{note}
 
 \pnum
-An atomic operation \term{A} that performs a release operation on an atomic
-object \term{M} synchronizes with an atomic operation \term{B} that performs
-an acquire operation on \term{M} and takes its value from any side effect in the
-release sequence headed by \term{A}.
+An atomic operation \placeholder{A} that performs a release operation on an atomic
+object \placeholder{M} synchronizes with an atomic operation \placeholder{B} that performs
+an acquire operation on \placeholder{M} and takes its value from any side effect in the
+release sequence headed by \placeholder{A}.
 
 \pnum
-There shall be a single total order \textit{S} on all \tcode{memory_order_seq_cst}
+There shall be a single total order \placeholder{S} on all \tcode{memory_order_seq_cst}
 operations, consistent with the ``happens before'' order and modification orders for all
 affected locations, such that each \tcode{memory_order_seq_cst} operation
-\textit{B} that loads a
-value from an atomic object \textit{M}
+\placeholder{B} that loads a
+value from an atomic object \placeholder{M}
 observes one of the following values:
 
 \begin{itemize}
-\item the result of the last modification \textit{A} of \textit{M} that precedes
-\textit{B} in \textit{S}, if it exists, or
+\item the result of the last modification \placeholder{A} of \placeholder{M} that precedes
+\placeholder{B} in \placeholder{S}, if it exists, or
 
-\item if \textit{A} exists, the result of some modification of \textit{M}
+\item if \placeholder{A} exists, the result of some modification of \placeholder{M}
 that is not
-\tcode{memory_order_seq_cst} and that does not happen before \textit{A}, or
+\tcode{memory_order_seq_cst} and that does not happen before \placeholder{A}, or
 
-\item if \textit{A} does not exist, the result of some modification of \textit{M}
+\item if \placeholder{A} does not exist, the result of some modification of \placeholder{M}
 that is not
 \tcode{memory_order_seq_cst}.
 \end{itemize}
 
-\begin{note} Although it is not explicitly required that \textit{S} include locks, it can
+\begin{note} Although it is not explicitly required that \placeholder{S} include locks, it can
 always be extended to an order that does include lock and unlock operations, since the
 ordering between those is already included in the ``happens before'' ordering. \end{note}
 
 \pnum
-For an atomic operation \textit{B} that reads the value of an atomic object \textit{M},
```

Comment:
Ah no worries, I'm glad to hear there's coherent plans being formed for the use of all these things going forward. :)  I'll split out the \term stuff tonight, cheers!